tasks/header.js

```diff
@@ -87,7 +87,10 @@ function updateHeadersInSrcFiles() {
     });
 
     function isCorrect(header) {
-        return (header.value === licenseStr);
+        return (
+            header.value.replace(/\s+$/gm, '') ===
```

Comment:
Is this for windows compat?

If so, I would prefer a different solution. Trimming space here could lead to false positive assertion.

Reply:
_never mind_. Thanks for the fix!